src/app.js

```diff
@@ -76,6 +77,7 @@ $(function () {
         cache: true,
         dataType: "script",
         success: () => {
+            var smartsupp
             smartsupp('on', 'status', status => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This expression might have a value which cannot be called; it might not be a function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2873)